packages/docs/pages/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
   "index": "Introduction",
+  "getting-started": "Getting Started",
```

Comment:
I think shorter script name would be more efficient. Something like "setup", "create", "clone" or "start"? What do you think?

Reply:
I thought it gives the user the feeling that this is the page where he can get started setting up Essencium for a new project. I was inspired by i.e. https://vitest.dev/guide

Reply:
Ah I see, I thought it's in a different file! Totally fine with the naming in _meta.json 😂